packages/account/build/constants.js

```diff
@@ -18,6 +18,7 @@ const {
 const IS_RELEASE = process.env.NODE_ENV === 'production' || process.env.NODE_ENV === 'staging';
 
 const ALIASES = {
+    Assets: path.resolve(__dirname, '../src/Assets'),
```

Comment:
@ali-hosseini-fs Don’t we want to get rid of import aliases in the future? 🤔

Reply:
I'm completely ok with it. but still we need other leads and devs opinion on this one.
The changes would be huge! almost all of the project files.